go-runtime/compile/build.go

```diff
@@ -145,6 +145,16 @@ var scaffoldFuncs = scaffolder.FuncMap{
 				}
 				imports[path.Join("ftl", n.Module)] = "ftl" + n.Module
 
+				for _, tp := range n.TypeParameters {
+					tpRef, err := schema.ParseDataRef(tp.String())
+					if err != nil {
+						return err
+					}
+					if tpRef.Module != "" && tpRef.Module != m.Name {
+						imports[path.Join("ftl", tpRef.Module)] = "ftl" + tpRef.Module
+					}
+				}
+
```

Comment:
@alecthomas I uncovered this with the tests. It seemed like imports for external modules in typeparameters weren't being imported. Does this fix look ok to you?

Reply:
Ah that's quite possible. Looks about right.